inference-engine/tests/functional/plugin/cpu/single_layer_tests/ctc_Loss.cpp

```diff
@@ -0,0 +1,245 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <functional_test_utils/ov_tensor_utils.hpp>
+#include <ngraph_functions/builders.hpp>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+
+using CTCLossShapeParams = std::pair<std::vector<ngraph::PartialShape>, std::vector<std::vector<ngraph::Shape>>>;
+
+using CTCLossLayerCPUTestParams = std::tuple<CTCLossShapeParams,            // dynamic, static shape[N, T, C]
+                                            bool,                           // preprocessCollapseRepeated
+                                            bool,                           // ctcMergeRepeated
+                                            bool,                           // unique
+                                            ElementType,                    // fp precision for logits
+                                            ElementType                     // int precision for label and length
+                                            >;
+ngraph::ParameterVector makeDynamicParams(const std::vector<ElementType>& types,
+                                          const std::vector<ov::PartialShape>& shapes) {
+    ngraph::ParameterVector outs;
+    NGRAPH_CHECK(types.size() == shapes.size());
+    for (size_t i = 0; i < types.size(); i++) {
+        auto paramNode = std::make_shared<ov::op::v0::Parameter>(types[i], shapes[i]);
+        outs.push_back(paramNode);
+    }
+    return outs;
+}
```

Comment:
better to place it inside `openvino/inference-engine/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/builders.hpp`

Reply:
moved to nGraph helper. Thanks